.github/workflows/pkgdown.yaml

```diff
@@ -42,6 +43,9 @@ jobs:
           remotes::install_github("r-lib/lifecycle")
         shell: Rscript {0}
 
+      - name: Set up license file
+        run: echo ${CONNECT_LICENSE_FILE} > $RSC_LICENSE
```

Comment:
Permissions are unlikely to matter in this context, but in case you want to see what we recommend: https://docs.posit.co/connect/admin/licensing/#license-file

Reply:
The file permissions look to be the same as what I have locally, which works. IDK about the user.

Reply:
After a bunch of attempts, it turns out I needed `echo "$CONNECT_LICENSE_FILE"` with quotes in order to correctly handle the newlines in the license file-in-env-var. This is working now.